cypress/support/affiliation.js

```diff
@@ -39,7 +44,12 @@ export function setWorkspaceInAffiliation(workspace = null, payerAuth = true) {
         response.body.configuration[payerAuthIndex].value = payerAuth
           ? 'active'
           : 'disabled'
-
+        cy.qe(`
+        curl --location --request PUT '${vtex.baseUrl}/api/payments/pvt/affiliations/21d78653-50d6-4b06-b553-2645e67a6f5e' \
```

Comment:
Replace curl with cy.request
if it is a common function then do it in cy-runner rather than here